DataFormats/Scouting/interface/Run3ScoutingElectron.h

```diff
@@ -86,15 +98,19 @@ class Run3ScoutingElectron {
   float eta() const { return eta_; }
   float phi() const { return phi_; }
   float m() const { return m_; }
-  float d0() const { return d0_; }
-  float dz() const { return dz_; }
```

Comment:
Based on https://github.com/cms-sw/cmssw/pull/41025#discussion_r1135582592 and replies, I would say these need to remain in place, so that `d0` and `dz` can be read from existing (2022) scouting data.